element/element.js

```diff
@@ -27,6 +27,7 @@ export default function element(selector) {
   }
 
   async function elementIsDisplayed() {
+    await el.waitForDisplayed();
     return el.isDisplayed();
   }
 
```

Comment:
isElementDisplayed should return value
If you need some logic for waiting without returned value just crete new function `waitForElementDisplayed`